app/models/yellow_ribbon_program.rb

```diff
@@ -1,4 +1,52 @@
 # frozen_string_literal: true
 
 class YellowRibbonProgram < ApplicationRecord
+  belongs_to :institution
+
+  validates :city, presence: true
+  validates :contribution_amount, numericality: true
+  validates :degree_level, presence: true
+  validates :division_professional_school, presence: true
+  validates :facility_code, presence: true
+  validates :number_of_students, numericality: true
+  validates :school_name_in_yr_database, presence: true
+  validates :state, presence: true
+  validates :street_address, presence: true
+  validates :zip, presence: true
```

Comment:
Validates only run for tables that are populated by CSV import as https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L449 uses pure SQL and does not trigger validates on objects

If you would like to validate that these fields have values then yellow_ribbon_program_source model needs the validates, however if you are going to ignore rows without these values at https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L449 then the validates are not needed

Reply:
https://guides.rubyonrails.org/active_record_validations.html#when-does-validation-happen-questionmark

Reply:
Hmm, so technically we can handle the scenario if all of these fields are missing on the frontend, so I think I can remove all of the validations tbh.